spec/classes/foreman_plugin_openscap_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper'
+
+describe 'foreman::plugin::openscap' do
+  on_supported_os.each do |os, facts|
+    if facts[:osfamily] == 'RedHat'
+      context "on #{os}" do
+        let :facts do
```

Comment:
you mean like this?
```ruby
describe 'foreman::plugin::openscap' do
  on_supported_os.each do |os, facts|
    let :facts do
      facts
    end

    if facts[:osfamily] == 'RedHat'
      context "on #{os}" do
      ...
```
This way they I'm getting errors since facts in spec seems OK while those used for catalog compilation seems to use cached value from another run, I suppose facts are cached in `describe` context in this case and can't be redeclared using let.

Also I think the question was more about whether we need 
```ruby
let (:facts) do 
  facts 
end
```

snippet even without customization, which I think we need, otherwise my facts seems empty.